src/main/java/tw/joi/energy/domain/ElectricityReading.java

```diff
@@ -1,9 +1,16 @@
 package tw.joi.energy.domain;
 
 import java.math.BigDecimal;
+import java.time.Clock;
 import java.time.Instant;
 
 /**
- * @param reading kWh
+ * @param time point in time
+ * @param readingInKwH energy consumed in total to this point in time in kWh
  */
-public record ElectricityReading(Instant time, BigDecimal reading) {}
+public record ElectricityReading(Instant time, BigDecimal readingInKwH) {
+
+    public ElectricityReading(Clock clock, double readingInKwH) {
```

Comment:
Is this constructor meant to be for testing purposes (faking the current time, lighter syntax for the reading) or also for production usage?